packages/perseus-editor/src/components/__stories__/graph-settings.argtypes.ts

```diff
@@ -50,7 +50,6 @@ export default {
             type: "select",
         },
         table: {
-            defaultValue: {summary: "graph"},
```

Comment:
Removed this everywhere because I learned that `defaultValue` under `table` is no longer the favored approach. Instead, set args directly in the stories.